hydromt_sfincs/sfincs.py

```diff
@@ -1011,6 +1016,126 @@ def setup_cn_infiltration(self, cn, antecedent_moisture="avg", reproj_method="me
         self.config.pop("qinf", None)
         self.set_config(f"{mname}file", f"sfincs.{mname}")
 
+    # Function to create curve number for SFINCS including recovery term (Kr)
+    def setup_curvenumber_infiltration_withrecovery(
+        self, data_catalog, name_files, reclass_table, effective, block_size=2000
+    ):
+        """Setup model the Soil Conservation Service (SCS) Curve Number (CN) files for SFINCS
+        including recovery term based on the soil saturation
+
+        Parameters
```

Comment:
We try to use the numpy docstring format, see also docstring of other methods. This style get's automatically rendered into nice html documentation. For more info see [hydromt dev guide](https://deltares.github.io/hydromt/latest/dev/contributing#docstrings)

Reply:
OK, sounds good!